src/ipc-handlers.ts

```diff
@@ -388,7 +388,17 @@ export async function startServer(
 	await keepSqliteIntegrationUpdated( server.details.path );
 
 	const parentWindow = BrowserWindow.fromWebContents( event.sender );
-	await server.start();
+	try {
+		await server.start();
+	} catch ( error ) {
+		if (
```

Comment:
Looks good. In the future, we could handle other exceptions there with custom messages, or maybe remove the condition and use unified message for all case.

Reply:
Agreed. I thought about overwriting all the errors, but I prefer to make the distinction in case we found any other error now or in the future after a dependency update.